saleor/tests/e2e/orders/test_cancel_fully_paid_order.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..product.utils.preparing_product import prepare_product
+from ..shop.utils import prepare_shop
+from ..transactions.utils import create_transaction
+from ..utils import assign_permissions
+from .utils import (
+    draft_order_complete,
+    draft_order_create,
+    draft_order_update,
+    order_cancel,
+    order_lines_create,
+    order_query,
+)
+
+
+@pytest.mark.e2e
+def test_cancel_fully_paid_order_CORE_0206(
+    e2e_staff_api_client,
+    e2e_app_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_shipping,
+    permission_manage_orders,
+    permission_manage_payments,
+):
+    # Before
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_product_types_and_attributes,
+        permission_manage_orders,
+        permission_manage_payments,
+    ]
+    assign_permissions(e2e_staff_api_client, permissions)
+    app_permissions = [permission_manage_payments, permission_manage_orders]
+    assign_permissions(e2e_app_api_client, app_permissions)
+
+    price = 10
+
+    (
+        result_warehouse_id,
+        result_channel_id,
+        _,
```

Comment:
I think you should use the variable name since this PR https://github.com/saleor/saleor/pull/13964 was merged